src/main/js/wiki/components/BranchDropdown.test.ts

```diff
@@ -0,0 +1,31 @@
+import { buildBranchUrl } from "./BranchDropdown";
+
+it("should replace one branch in the url with a new one", () => {
```

Comment:
This test doesn't work on my machine. It throws an error: `Cannot convert undefined or null to object`
Furthermore it seems like it is not executed by CI

Reply:
I don't seem to have found a suitable way to mock the URL at this point, would it be ok if I deleted the test?

Reply:
replaced the function that is tested which makes the test obsolete